homeassistant/components/speedtestdotnet/__init__.py

```diff
@@ -22,61 +18,14 @@
     DEFAULT_SCAN_INTERVAL,
     DEFAULT_SERVER,
     DOMAIN,
-    SENSOR_TYPES,
+    PLATFORMS,
     SPEED_TEST_SERVICE,
 )
 
 _LOGGER = logging.getLogger(__name__)
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
I saw that the config has been deprecated starting 2021.6. So when can we remove it? 
Also If we keep the deprecated schema does that mean it will still be imported?

Reply:
You can still remove it, just leave the single deprecation line.